src/v/kafka/server/group_tx_tracker_stm.h

```diff
@@ -133,12 +134,14 @@ class group_tx_tracker_stm final
     }
 
 private:
-    static constexpr int8_t supported_local_snapshot_version = 1;
+    static constexpr int8_t supported_local_snapshot_version = 2;
```

Comment:
this doesn't need a bump I think. (It should only be bumped if the snapshot should be discarded after upgrade)